config/samples/ruby_example_loadtest_with_pre_built_workers.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: prebuilt-ruby-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: ruby
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+  - language: ruby
+    run:
+      image: gcr.io/grpc-testing/donghao/prebuilt/ruby
```

Comment:
We don't want to mention the explicit cloud project 'grpc-testing' and folder 'donghao' here. This sample will not be runnable by anyone who does not have GCR read access here. This should be made into a fake image path, or it should be used to reopen discussions about making our container images public. /cc @grpc/test-infra

Reply:
I would change these files into templates. In this case, the location could be ${prebuilt_image_prefix} of some such. I would use Python template notation, since these templates will be processed by loadtest_template.py in grpc/grpc (compare that with the templates in https://github.com/grpc/test-infra/blob/master/config/defaults_template.yaml, which are processed by a go script from our makefiles.

If we make these files into templates, I would create a new folder config/samples/templates for them, and perhaps add a README.md there explaining that these are templates and should be run through loadtest_template.py first.

I would ultimately run all languages through loadtest_template.py and create a new master template in grpc/grpc, similar to the one in https://github.com/grpc/grpc/pull/25976.

Reply:
done.